src/webots/gui/WbNewProtoWizard.cpp

```diff
@@ -141,9 +142,19 @@ void WbNewProtoWizard::accept() {
             tokenizer.tokenizeString(parameters[i]);
             WbParser parser(&tokenizer);
             foreach (const QString &node, parser.protoNodeList()) {
-              QString url = WbProtoManager::instance()->protoUrl(node, mCategory);
+              const QString parentUrl = url.replace("webots://", WbStandardPaths::webotsHomePath());
+              QString nestedUrl = WbProtoManager::instance()->findExternProtoDeclarationInFile(parentUrl, node);
+              // Replace local URL of nested nodes in distributed remote parent nodes
+              const QString prefix = WbUrl::computePrefix(parentUrl);
+              if (!prefix.isEmpty() && (WbUrl::isLocalUrl(nestedUrl) || QDir::isRelativePath(nestedUrl))) {
```

Comment:
I haven't tested to see if it's actually an issue in practice, but I just want to point out that if `nestedUrl` happens to be remote (`http://...`), then the `QDir::isRelativePath` activates as well and I don't believe in this case we wish to execute the body of the function. Or rather, if it's already a web-url, we want to use it without any manipulation. In short, I think a condition `WbUrl::isWeb(nestedUrl)` prior to doing anything else might be missing